proxybroker/negotiators.py

```diff
@@ -92,6 +95,9 @@ async def negotiate(self, **kwargs):
 
         await self._proxy.send(struct.pack('>2BH5B', 4, 1, port, *bip, 0))
         resp = await self._proxy.recv(8)
+        if isinstance(resp, asyncio.Future):
```

Comment:
Have you encountered any issues here?
Because the response comes from `asyncio` and it should only possibly be the `Future` object.